modules/integration/tests-ui-integration/src/test/java/org/wso2/carbon/mb/ui/test/dlc/DLCDurableTopicTestCase.java

```diff
@@ -174,55 +187,50 @@ public void addDurableTopicMessagesToDLC()
     @Test()
     public void performDurableTopicDeadLetterChannelTestCase() throws IOException {
 
+        // Number of checks for an update in consumer message count.
+        int tries = 15;
```

Comment:
Theres nothing special. Earlier there was an infinite loop checking for changes in message count. Therefore I used a timeout count. Else the test case would get stuck if the UI flow is broken.